test/inductor/test_torchinductor.py

```diff
@@ -800,6 +800,79 @@ def repeat(x, n):
         self.assertEqual(expect, actual)
         self.assertEqual(actual, repeat(x, 3))
 
+    @skipIfRocm
+    def test_neg_index(self):
+        def test(fn, inps, has_assert: bool, has_wrapping=True):
+            for dynamic in (True, False):
+                fn_opt = torch.compile(dynamic=dynamic)(fn)
+                if self.device == "cpu":
+                    code = run_and_get_cpp_code(fn_opt, *inps)
+                    lines = code.split("\n")
+                    found = False
+                    if "Vectorize" not in code:
+                        for line in lines:
+                            # match ternary operator
+                            pattern = r"\?.*:"
+                            if re.findall(pattern, line):
```

Comment:
I mean, you can do that in the whole code in one go, right?

Reply:
Yes, right.